pkg/base/k8s/kubeclient.go

```diff
@@ -345,6 +345,25 @@ func (k *KubeClient) GetNodes(ctx context.Context) ([]coreV1.Node, error) {
 	return nodes.Items, nil
 }
 
+// GetSystemDriveUUID returns system drives uuid
+// Receives golang context
+// Returns return slice of string - system drives uuid
+func (k *KubeClient) GetSystemDriveUUID(ctx context.Context) []string {
+	ll := k.log.WithField("method", "GetSystemDriveUUID")
+	var driveList drivecrd.DriveList
+	if err := k.ReadList(ctx, &driveList); err != nil {
+		ll.Errorf("Failed to get system disk uuid, error: %v", err)
+		return nil
+	}
+	driveUUID := make([]string, 0)
+	for _, drive := range driveList.Items {
```

Comment:
Do we need to return error hen drive not found?

Reply:
Do you mean return an error if there are no system drives were found in `driveList`? In this case we can log error and return the empty slice.

Reply:
Yes, please. It will help with the troubleshooting.

Reply:
Fixed